docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -617,6 +617,66 @@ It is possible to use multiple expressions in the role definition.
 In development mode, it allows any authenticated user.
 <5> Property expression `all-roles` will be treated as a collection type `List`, therefore, the endpoint will be accessible for roles `Administrator`, `Software`, `Tester` and `User`.
 
+=== Endpoint security annotations and Jakarta REST inheritance
+
+Quarkus supports security annotations placed on the endpoint implementation or its class like in the example below:
+
+[source,java]
+----
+@Path("hello")
```

Comment:
Michal, just to clarify, we won't require  `@Path` be set on the implementation for the inheritance to work, right ?

Reply:
No, purpose of this example was to show what is meant by "endpoint implementation". I am happy to adjust it. Please shoot suggestions.

Reply:
In other words, we don't have influence on what RESTEasy marks as Resource class (or declaring class of Resource method). We only care about where is declared method that is actually invoked, regardless of JAX-RS endpoint defining annotations. The actually invoked method is the one that "returns something".

Reply:
@michalvavrik Makes sense